data/tlg4034/tlg006/__cts__.xml

```diff
@@ -2,6 +2,6 @@
   <ti:title xml:lang="eng">In Librum Quintum Ethicorum Nicomacheorum Commentarium</ti:title>
```

Comment:
The `xml:lang` attribute on the `<ti:title>` element should be "lat" since the title is in Latin.